### DISCUSSION TITLE
nextcloud-aio-borgbackup container created in test mode

### DISCUSSION THREAD

**Discussion by smaluish** [nextcloud/all-in-one#4904] [Questions] [answered]
Restoring new instance of Nextcloud on a new machine using NFS folder from shared nas, the backup is located and is checked for integrity and passes.
Proceed to restore from backup and everything runs fine all data is restored and containers are started up and i am able to log into nextcloud.
If i try to start a backup it fails as there is no longer a nextcloud-aio-borgbackup container.
After resetting the master container and trying the process again I have checked the nextcloud-aio-borgbackup container and noticed it is in test mode, again once the backup is completed the nextcloud-aio-borgbackup container is not recreated with the other containers.
Is this normal and if so how do i recreate the nextcloud-aio-borgbackup container.

**Comment by szaimen** [comment, marked as answer]
> If i try to start a backup it fails as there is no longer a nextcloud-aio-borgbackup container

Hi, can you please clarify how you started the backup? Did you use the `Create backup` button via the AIO interface?

**Reply by smaluish** [reply]
Yes I would use the `Create backup` button via the AIO interface.
I have read the logs again and I think it is failing on the `additional directories` I had in the backup and the directory had not been created for the backup to copy files to. I have since removed the `additional directories` and it is working fine now.
Would this stop the `nextcloud-aio-borgbackup` container from being created?

**Reply by szaimen** [reply]
> Would this stop the `nextcloud-aio-borgbackup` container from being created?

Yes
